internal/util/errors_test.go

```diff
@@ -30,42 +30,3 @@ var (
 func wrapError(e error) error {
 	return fmt.Errorf("w{%w}", e)
 }
-
```

Comment:
@Thingee changes looks good to be, we can remove this file as we don't need `wrapError` not `errFoo` and `errBar` anymore as they are not used.